packages/typeahead/tests/__snapshots__/Typeahead.test.tsx.snap

```diff
@@ -1,6 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Typeahead renders 1`] = `
+.emotion-3 {
```

Comment:
Why are we updating the typeahead snapshots? Did we touch anything here?

Reply:
We didn't. I have no idea why the snapshots changed - they styles look the same